diagnostic.md

```diff
@@ -94,4 +94,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+_hobby is exercise name is jason and surname is weeks._
```

Comment:
teamMember['name'] refers to the object { given: 'Jason', surname: 'Weeks' }.  This is what is known as a "nested object" since it's an object within another object.  Remember that strings